From 9c141a3b230f1e2e5ecdb2eb95ad597610df73cd Mon Sep 17 00:00:00 2001
From: Sjoerd Simons <sjoerd@luon.net>
Date: Fri, 20 Jul 2012 21:18:26 +0200
Subject: [PATCH] webrtc-utils needs to be installed before module-echo-cancel

Like other plugin helper libraries these need to be installed
before the plugins that use them..
---
 src/Makefile.am |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 9b71b17..aae13b5 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -905,6 +905,10 @@ modlibexec_LTLIBRARIES = \
 		libprotocol-http.la \
 		libprotocol-native.la
 
+if HAVE_WEBRTC
+modlibexec_LTLIBRARIES += libwebrtc-util.la
+endif
+
 if HAVE_ESOUND
 modlibexec_LTLIBRARIES += \
 		libprotocol-esound.la
@@ -1837,7 +1841,6 @@ if HAVE_WEBRTC
 # The webrtc code is split off into a helper library to avoid having automake
 # link module-echo-cancel with C++ (which it does if there are any C++ deps,
 # even conditional ones).
-modlibexec_LTLIBRARIES += libwebrtc-util.la
 
 libwebrtc_util_la_SOURCES = modules/echo-cancel/webrtc.cc
 libwebrtc_util_la_CXXFLAGS = $(AM_CXXFLAGS) $(SERVER_CFLAGS) $(WEBRTC_CFLAGS) -DHAVE_WEBRTC=1
-- 
1.7.10.4

